cartridges/int_bolt_embedded_sfra/cartridge/client/default/js/account.js

```diff
@@ -30,13 +30,11 @@ async function authorize(customerEmail) {
 }
 
 /**
- * Log the user into their bolt account
- * @param {string} email - input email
+ * Auto log the user into their bolt account
  * @returns {Promise} The returned promise to fetch account details
  */
-async function login(email) {
-    const authorizeResp = await authorize(email);
-    $('.submit-customer').removeAttr('disabled'); // enable checkout button after OTP modal is rendered
```

Comment:
Move this line because this function is now for auto login and there will be no need to enable the button